internal/k8s/controller.go

```diff
@@ -153,9 +154,105 @@ func isRouteUpdated(oldRoute, newRoute *routev1.Route) bool {
 	return false
 }
 
+func isNamespaceUpdated(oldNS, newNS *corev1.Namespace) bool {
+	if oldNS.ResourceVersion == newNS.ResourceVersion {
+		return false
+	}
+	oldLabelHash := utils.Hash(utils.Stringify(oldNS.Labels))
```

Comment:
if I have labels like this:
 app:prod
and 
 prod:app

I presume, they will be two different hashes?

Reply:
it wll be different